### PR TITLE
Fixes for several reported issues

### PR DESCRIPTION
**Pull request by seriva**
- Added missing template for disk_size_gb storage_os_disk (#1043)
- Add ability to add subsciptionId to sp.yml on Azure (#1044)
- Added Epicli binary and python decencies/components.
- Updated documentation about sp.yml to show usage of " for values that have YAML schema characters.

### CODE REVIEW

File: core/src/epicli/cli/engine/terraform/TerraformRunner.py [hunk 1]
```diff
@@ -49,22 +49,30 @@ def azure_login(self):
                 subscription = apiproxy.login_account()
                 apiproxy.set_active_subscribtion(subscription['id'])
 
-                # Create the service principle
+                # Create the service principle, for now we use the default subscription
```
Comment (erzetpe) [inline core/src/epicli/cli/engine/terraform/TerraformRunner.py]: Should be service principal

Comment (seriva) [reply core/src/epicli/cli/engine/terraform/TerraformRunner.py]: Out of principle I will change it!

File: CHANGELOG-0.6.md [hunk 2]
```diff
@@ -8,8 +8,10 @@
 - [#987](https://github.com/epiphany-platform/epiphany/issues/987) - Add verbosity levels for Terraform and Ansible
 - [#656](https://github.com/epiphany-platform/epiphany/issues/656) - Add logrotation to kafka by size
 - [#1016](https://github.com/epiphany-platform/epiphany/issues/1016) - Disable verify , backup and recovery as they are not fully implemented
+- [#1044](https://github.com/epiphany-platform/epiphany/issues/1044) - Add ability to add subsciptionId to sp.yml on Azure
```
Comment (plirglo) [inline CHANGELOG-0.6.md]: subsciptionId - typo

File: core/src/epicli/gen-licenses.py [hunk 3]
```diff
@@ -90,5 +93,26 @@ def _main() -> None:
     with open(path, 'w') as file:
         file.write(licenses_content)
 
+     # Write components table to be pasted into 'COMPONENTS.md'
+    dependancies_content = """
```
Comment (plirglo) [inline core/src/epicli/gen-licenses.py]: typo - but should work

File: core/src/epicli/gen-licenses.py [hunk 4]
```diff
@@ -90,5 +93,26 @@ def _main() -> None:
...
+        dep_name = dep['Name']
+        dep_version = dep['Version']
+        dep_website = dep['Home-page']
+        dep_license = dep['License']
+        if 'License URL' in dep:
+            dep_license_url = dep['License URL']
+            dep_line = f'| {dep_name} | {dep_version} | {dep_website} | [{dep_license}]({dep_license_url}) |\n'
+        else:
+            dep_line = f'| {dep_name} | {dep_version} | {dep_website} | {dep_license} |\n'
+        dependancies_content = dependancies_content + dep_line
```
Comment (plirglo) [inline core/src/epicli/gen-licenses.py]: same as above

File: core/src/epicli/gen-licenses.py [hunk 5]
```diff
@@ -90,5 +93,26 @@ def _main() -> None:
...
+
+    path = os.path.join(os.path.dirname(__file__), 'DEPENDENCIES.md')
+    with open(path, 'w') as file:
+        file.write(dependancies_content)
```
Comment (plirglo) [inline core/src/epicli/gen-licenses.py]: same as above

**Pull request merged**